.github/workflows/ci.yml

```diff
@@ -4,39 +4,29 @@ on: [push, pull_request, workflow_dispatch]
 
 jobs:
   tests:
-    name: Tests with Ruby ${{ matrix.ruby-version }} and Rails ${{ matrix.gemfile }}
+    name: CI Rails ${{ matrix.rails }} on ${{ matrix.ruby }}
```

Comment:
I am making this shorter, so its easier to see the version on Actions details.
![image](https://github.com/mileszs/wicked_pdf/assets/99191/aa240650-f0ce-45e7-969c-2f6023b87ec8)